guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -24,6 +24,11 @@ ifdef::satellite[]
 * {Project} {ProjectVersion} running on {RHEL} 7.
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you run {Project} in a disconnected environment, you must obtain and deploy Leapp metadata manually. For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+* You require access to {RHEL} and {Project} packages.
+* Set `module_hotfixes=1` in the repository file when setting up the {Project} repositories.
+* When running `leapp preupgrade` and `leapp upgrade`, you must pass `--enablerepo <id>` for each manually defined repository from the previous step.
```

Comment:
Being a bit blunt, I do not know what I need to pass to `--enablerepo <id>`. This is a bit unspecific and for that matter the set of instructions is quite vague. Could we beef this up by:

 1) Pointing users where and how to get the RHEL 8 ISO and Satellite on RHEL 8 ISOs
 2) Instruct users how to unpack them
 3) Make `--enablerepo <id>` specific with each repository given we are instruction the user on how to setup up the RHEL 8 and Satellite on RHEL 8 repositories from the ISOs

I would be OK if this re-uses or points back to the disconnected guide that has these detailed instructions (e.g. https://github.com/theforeman/foreman-documentation/pull/1425).

My goal here would be to ensure we are as straight forward about the process as we can be to help our users out and prevent unnecessary support tickets.

Reply:
@ehelms Take a look at the latest change and see if these steps make sense. I added a reference to ISO files in the Upgrading a Disconnected Satellite guide before discussing the `enablerepo <id>`. Change the wording and order of the steps and/or add any additional content you think is necessary.